src/components/common/copyable/copyable-field/copyable-field.tsx

```diff
@@ -28,7 +31,7 @@ export const CopyableField: React.FC<CopyableFieldProps> = ({ content, nativeSha
         url: url
       })
       .catch((err) => {
-        console.error('Native sharing failed: ', err)
+        log.error('Native sharing failed: ', err)
```

Comment:
We should either include the `: ` for each error log message or leave it out everywhere.
I personally would remove it everywhere.

Reply:
Me too. Seems like I missed this part